Classes/Domain/CacheConfigurationVersionProviderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Ui\Domain;
```

Comment:
shouldnt we group all these `ProviderInterface`s into one sub namespace like `Neos\Neos\Ui\Domain\BackendProviders`?

That way `Neos\Neos\Ui\Domain` is not that cluttered and there is also "space" for other interfaces not directly related to the backend, like the NodeCreationHandlerInterface for example?

I could take care of that refactoring ^^

Reply:
Or maybe `Neos\Neos\Ui\Domain\InitialData` or `Neos\Neos\Ui\Domain\Booting` or `Neos\Neos\Ui\Domain\InitialDataProvider`?

Reply:
Discussed this with Wilhelm and he was okay with moving it to `InitialData`.